packages/react/src/feed/FeedEventItemType.ts

```diff
@@ -0,0 +1,9 @@
+import { FeedEventItemType as LensFeedEventItemType } from '@lens-protocol/api-bindings';
+
+export enum FeedEventItemType {
```

Comment:
nit: just thinking, maybe it's not the responsibility of `@lens-protocol/react` to protect which `FeedEvenItemType` is supported but rather `@lens-protocol/react` should overwrite the `useFeedHook` to guide what can be used?

Reply:
after thinking a little bit more it seems not worth the effort to add more complexity to the `api-bindings`